languageFactories/language-factory-core/src/main/scala/cromwell/languages/util/LanguageFactoryUtil.scala

```diff
@@ -94,7 +94,7 @@ object LanguageFactoryUtil {
     }
 
     val firstCodeLine = fileWithoutInitialWhitespace.headOption.map(_.dropWhile(_.isWhitespace))
-    firstCodeLine.exists { line => startsWithOptions.exists(line.startsWith) }
+    firstCodeLine.exists { line => startsWithOptions.contains(line) }
```

Comment:
Does this catch things we don't want, like ?
```
# Thank goodness this WDL is not version 1.1!
```

Reply:
The `commentIndicators` value is defined as `List("#")` and appears to drop comments from consideration.